keyboards/mexsistor/ludmila/ludmila.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2020 Kevin M.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+#define ___ KC_NO
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
+    k00, k01, k20, \
+    k10, k11 \
+) { \
+    { k00, k01, }, \
+    { k10, k11, }, \
+    { k20, ___, }  \
```

Comment:
```suggestion
    { k00, k01 }, \
    { k10, k11 }, \
    { k20, ___ }  \
```

Reply:
There is no special reason to leave that part there, I use it as a kind of "mapping" when assigning my macros. But you can remove it if you want :) Thank you for your review and your comments.